src/sentry/web/frontend/remove_organization.py

```diff
@@ -61,6 +56,21 @@ def handle(self, request, organization):
                     countdown=86400,
                 )
 
+                self.create_audit_entry(
```

Comment:
It's in both. https://github.com/getsentry/sentry/blob/master/src/sentry/web/frontend/base.py#L252-L263